### PR TITLE
NativeConstantInvocationFixer - add "strict" flag

### PR DESCRIPTION
**Pull request by kubawerlos**
@nicolas-grekas is it OK to add it to `@Symfony` rule set (I did it as it is very similar "strict" as for `NativeFunctionInvocationFixer`)?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -956,6 +956,8 @@ Choose from the list of available rules:
   - ``include`` (``array``): list of additional constants to fix; defaults to ``[]``
   - ``scope`` (``'all'``, ``'namespaced'``): only fix constant invocations that are made
     within a namespace or fix all; defaults to ``'all'``
+  - ``strict`` (``bool``): whether leading ``\`` of constant call not meant to have it
```
Comment (SpacePossum) [inline README.rst]: if would be prefer `whether leading ``\`` of constant usage not meant to have it should be removed` as constants are not really called, WDYT?

**Comment by SpacePossum** [comment]
Im very much :+1:  for adding to the SF set

**Comment by nicolas-grekas** [comment]
Me too: `\` should be removed on constants that don't need it.

File: src/Fixer/ConstantNotation/NativeConstantInvocationFixer.php [hunk 2]
```diff
@@ -225,14 +225,18 @@ protected function createConfigurationDefinition()
                 ->setAllowedValues(['all', 'namespaced'])
                 ->setDefault('all')
                 ->getOption(),
+            (new FixerOptionBuilder('strict', 'Whether leading ``\`` of constant invocation not meant to have it should be removed.'))
```
Comment (julienfalque) [inline src/Fixer/ConstantNotation/NativeConstantInvocationFixer.php]: `` ` `` should not be doubled here (they get tripled in the README file).

Comment (kubawerlos) [reply src/Fixer/ConstantNotation/NativeConstantInvocationFixer.php]: Of course 🤦‍♂ - fixed.

**Comment by SpacePossum** [comment]
Thank you @kubawerlos.

**Pull request merged**